### ISSUE TITLE
Bug: installation fails on Windows 10/R 3.4.3

### ISSUE THREAD

**Issue by chiffa** [drisso/archive-zinbwave#47]
Following the installation instructions provided on the [BioConductoR website](https://bioconductor.org/packages/release/bioc/html/zinbwave.html),

```
source("https://bioconductor.org/biocLite.R")
biocLite("zinbwave")
```

Results in the following error in Windows 10, R version 3.4.3: 

```
installation path not writeable, unable to update packages: MASS, mgcv, nlme, rpart
```

Which in turn prevents the package from properly installing.

**Comment by jpvert** [comment]
This is not a bug of zinbwave, but looks like an issue with your local file system. The error message says that it can not update the packages because it does not have the right to write in some directory. If you have a system administrator, maybe you should ask her/him; otherwise check the numerous discussions on this topic at https://www.google.fr/search?q=r+installation+path+not+writable , for example https://github.com/RevolutionAnalytics/RRO/issues/144 
In short, if you are your own administrator, then you could install the packages by running R as the administrator; if you are not your administrator, then you could ask your administrator to do it for you; otherwise, you should install R packages in a local directory where you have the right to write, by changing some environment variables.

**Comment by chiffa** [comment]
Thanks - I am coming from a python background and am installing R for this library specifically, so apologies for lacking the basics. The fact that some dependencies were able to update but not others was not making much sense to me.
